glide-core/src/protobuf/redis_request.proto

```diff
@@ -136,6 +136,7 @@ enum RequestType {
     DBSize = 92;
     Brpop = 93;
     Hkeys = 94;
+    HRandField = 95;
```

Comment:
I thought we were supposed to take the next available number on main since it doesn't seem feasible to check all open PRs to see which numbers they are using. What's the suggested approach to determining which number to use?

Reply:
I think ~117 is the next free one. Pick any, in the worst case you will update it when resolving conflicts

Reply:
Sorry I'm confused, is it fine to use 95 (in other words, pick any available one and then update it when resolving conflicts) or should I be somehow checking which next one is free?

Reply:
95 is already busy on upstream
You can keep it for now and update later when you'll be resolving conflicts vs upstream/main